config.md

```diff
@@ -127,4 +127,77 @@ For Linux-based systems the user structure has the following fields:
 Interpretation of the platform section of the JSON file is used to find which platform-specific sections may be available in the document.
 For example, if `os` is set to `linux`, then a JSON object conforming to the [Linux-specific schema](config-linux.md) SHOULD be found at the key `linux` in the `config.json`.
 
+## Configuration Schema Example
+
+Here is a full exmample `config.json` contents for reference (generated with `runc spec`).
+
+```json
+{
+	"version": "0.2.0",
+	"platform": {
+		"os": "linux",
+		"arch": "amd64"
+	},
+	"process": {
+		"terminal": true,
+		"user": {
+			"uid": 0,
+			"gid": 0,
+			"additionalGids": null
+		},
+		"args": [
+			"sh"
+		],
+		"env": [
+			"PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
+			"TERM=xterm"
+		],
+		"cwd": ""
+	},
+	"root": {
+		"path": "rootfs",
+		"readonly": true
+	},
+	"hostname": "shell",
+	"mounts": [
+		{
+			"name": "proc",
+			"path": "/proc"
+		},
+		{
+			"name": "dev",
+			"path": "/dev"
+		},
+		{
+			"name": "devpts",
+			"path": "/dev/pts"
+		},
+		{
+			"name": "shm",
+			"path": "/dev/shm"
+		},
+		{
+			"name": "mqueue",
+			"path": "/dev/mqueue"
+		},
+		{
+			"name": "sysfs",
+			"path": "/sys"
+		},
+		{
+			"name": "cgroup",
+			"path": "/sys/fs/cgroup"
+		}
+	],
```

Comment:
Excepting mqueue and cgroup, these are all covered by #164, so I'd leave the covered entries out (see mrunalp/ocitools#15).